content/en/contribute/code/core/build-commands.md

```diff
@@ -0,0 +1,66 @@
+
+## Development build commands
+
+For developers (humans) to execute to build `cht-core`.
+
+| Command                       | Description |
+| ----------------------------- | -- |
+| `build-ddocs`                 | Compiles all the ddocs and outputs them into `/api/build/ddocs` ready for deployment |
+| `build-dev`                   | Updates dependencies and builds all the applications. |
+| `build-dev-watch`             | Same as `build-dev`, but keeps watching for any code changes and automatically deploys on change. |
+| `build-documentation`         | Executes jsdoc on all the applications. |
+| `build-webapp-dev`            | Compiles the `/webapp` application. |
+| `copy-api-resources`          | Copies the static api files into the `api` build directory ready for deployment. |
+| `dev-api`                     | Sets up and runs the `api` server, and automatically deploys source changes. |
+| `dev-sentinel`                | Sets up and runs the `sentinel` server, and automatically deploys source changes. |
+| `local-images`                | Builds the docker images and updates the docker compose files. |
+| `update-service-worker`       | Updates the service worker file for deployment. |
+
+## Development test commands
+
+For developers to execute to test `cht-core`.
+
+| Command                       | Description |
+| ----------------------------- | -- |
+| `default-wdio-mobile-local`   | Compiles the app and executes the mobile e2e test suite. |
+| `e2e-integration-local`       | Compiles the app and executes the e2e integration test suite. |
+| `integration-api`             | Compiles the app and executes the api integration test suite. |
+| `lint`                        | Perform static analysis checks on the codebase. |
+| `standard-wdio-local`         | Compiles the app and executes the standard e2e test suite. |
+| `test`                        | Same as running `lint`, `unit`, and `integration-api`. |
+| `unit`                        | Executes unit test suites for all applications. |
+| `unit-admin`                  | Executes the unit test suite on `admin`. |
+| `unit-api`                    | Executes the unit test suite on `api`. |
+| `unit-sentinel`               | Executes the unit test suite on `sentinel`. |
+| `unit-shared-lib`             | Executes the unit test suite on all `shared-lib` modules. |
+| `unit-webapp`                 | Executes the unit test suite on `webapp`. |
+| `unit-webapp-continuous`      | Executes the unit test suite on `webapp`, and re-runs on code change. |
+| `wdio-local`                  | Compiles the app and executes the default e2e test suite. |
+
+## CI commands
+
+For Continuous Integration (robots) to run to build and test `cht-core`.
+
+| Command                       | Description |
+| ----------------------------- | -- |
+| `build`                       | Compiles, minifies, bundles the code, and builds the DDocs for publishing. |
```

Comment:
Above, you use `ddocs`, and here you use `DDocs`. If it is the same thing, please use one of them for consistency.